README.md

```diff
@@ -13,7 +13,7 @@ Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.
 Once Helm is set up properly, add the repo as follows:
 
 ```console
-helm repo add influxdata https://helm.influxdata.com
+$ helm repo add influxdata https://helm.influxdata.com/
```

Comment:
As per https://github.com/updownpress/markdown-lint/blob/master/rules/014-commands-show-output.md, dollar signs should not precede shell commands if no output is shown. Could we remove them from all such cases?

Reply:
Ah ok, it was to be consistent with all other code samples ; I'll fix all of them then now I know the rule :)

Reply:
Fixed !